modules/ROOT/pages/provisioning-virtualbox.adoc

```diff
@@ -0,0 +1,191 @@
+
+- Download the latest ISO image from the https://getfedora.org/coreos/download?tab=metal_virtualized&stream=stable[download page] or with podman (see https://coreos.github.io/coreos-installer/cmd/download/[documentation] for options):
+[source, bash]
+----
+podman run --security-opt label=disable --pull=always --rm -v .:/data -w /data \
+    quay.io/coreos/coreos-installer:release download -f iso
+----
+
+- Create a sufficiently large disk and attach it to the previously created VM:
+[source, bash]
+----
+FCOS_DISK=$(realpath ~/Downloads/fedora-coreos-34.20210711.3.0-live.x86_64.iso)
+
+vboxmanage createmedium --filename "${VM_DISK}" --size 10240 --format VDI
+vboxmanage storageattach "${VM_NAME}" --storagectl SATA --type hdd --port 0 --device 0 --medium "${VM_DISK}"
+----
+
+[NOTE]
+====
+*Fedora CoreOS requires the root filesystem to be at least 8 GiB.* For practical reasons, disk images for some platforms ship with a smaller root filesystem, which by default automatically expands to fill all available disk space. If you add additional partitions after the root filesystem, you must make sure to explicitly resize the root partition as shown so that it is at least 8 GiB.
+
+Currently, if the root filesystem is smaller than 8 GiB, a warning is emitted on login. Starting from June 2021, if the root filesystem is smaller than 8 GiB and is followed by another partition, Fedora CoreOS will refuse to boot. For more details, see https://github.com/coreos/fedora-coreos-tracker/issues/586[this bug].
+====
+
+- Attach a DVD drive and the live ISO to the VM:
+[source, bash]
+----
+vboxmanage storagectl "${VM_NAME}" --name IDE --add ide --controller PIIX4 --hostiocache on --portcount 2 --bootable on
+vboxmanage storageattach "${VM_NAME}" --storagectl IDE --type dvddrive --port 1 --device 0 --medium "${FCOS_DISK}"
+----
+
+You can now boot the target system you have configured. The ISO is capable of bringing up a fully functioning FCOS system purely from memory (i.e. without using any disk storage). Once booted, you will have access to a bash command prompt.
+
+You can now run `coreos-installer` inside the VM:
+[source, bash]
+----
+sudo coreos-installer install /dev/sda \
+    --insecure-ignition --ignition-url http://10.0.2.2:8000/example.ign
+----
+
+TIP: Check out `coreos-installer install --help` for more options on how to install Fedora CoreOS.
+
+Once the installation is complete, you have to remove the live ISO after shutting down (`sudo shutdown -h now`) the VM. Once the ISO is removed, you can start the VM anew. The actual first boot process begins now. It is at this time that Ignition ingests the configuration file and provisions the system as specified.
+
+[source, bash]
+----
+vboxmanage storageattach "${VM_NAME}" --storagectl IDE --type dvddrive --port 1 --device 0 --medium emptydrive
+vboxmanage startvm "${VM_NAME}" --type headless
+# Once the VM is available, you can log in:
+ssh core@localhost -p 2222
+----
+
+== Installing from Raw
```

Comment:
I appreciate the work that went behind writing and testing this. But personally I'm hesitant to document it as if it were a supported workflow. I think to start, just having the ISO install path would be great. Or otherwise if folks really want, maybe we should at least have a disclaimer around here that this is not heavily tested and may break.

Reply:
That somewhat mirrors my own thoughts, in particular concerning the actual locations of the config file, disk name etc. At the same time this process works spectacularly well (imho better than the regular ISO install with the cumbersome "press e at the right time" issue) if the user is already on Linux. Also, technically I should've probably used `parted` instead of `fdisk` but in the end it probably doesn't matter too much.
I am definitely in favor of the disclaimer, though.

Reply:
Let's make a big warning and keep it? It's also already mentioned in a second time after the ISO. I understand it's a bit fragile but it's a nice performance improvement for non casual users.

Otherwise we could split this into 2 PRs to merge the first part and make a new PR to discuss improvements for the raw disk part.

Reply:
Yup sure, I'm OK with keeping it and just adding a warning.

Reply:
I updated the pull request with such a warning.

Reply:
```suggestion
== Installing from raw bare metal image
```